internal/test/wsu/wsu_test.go

```diff
@@ -111,6 +115,35 @@ func TestWSU(t *testing.T) {
 	t.Run("Network annotations were applied to node", testHybridOverlayAnnotations)
 	t.Run("HNS Networks were created", testHNSNetworksCreated)
 	t.Run("East-west networking", testEastWestNetworking)
+	t.Run("Check cni config generated on the Windows host", testCNIConfig)
```

Comment:
nit: This test should be done before the `East-west networking` test, as the tests are currently being run in a order where the tested elements build on top of the previous test.